docs/how-to-guides/sync-dicom-metadata-to-fhir.md

```diff
@@ -1,12 +1,90 @@
+
+### Create a resource application for FHIR and DICOM servers
+
+For both your FHIR and DICOM servers, you will create a resource application in Azure. Follow the instructions below for each server, once for your Medical Imaging Server for DICOM and once for your FHIR Server.
+
+1. Sign into the [Azure Portal](https://ms.portal.azure.com/). Search for **App Services** and select the FHIR or DICOM App Service. Copy the **URL** of the App Service.
+1. Select **Azure Active Directory** > **App Registrations** > **New registration**:
+    1. Enter a **Name** for your app registration.
+    2. In **Redirect URI**, select **Web** and enter the **URL** of your App Service.
+    3. Select **Register**.
+1. Select **Expose an API** > **Set**. You can specify a URI as the **URL** of your app service or use the generated App ID URI. Select **Save**.
+1. Select **Add a Scope**:
+    1. In **Scope name**, enter *user.impersonation*.
+    1. In the text boxes, add an admin consent display name and admin consent description you want users to see on the consent page. For example, *access my app*.
+
+### Set the Authentication for your FHIR & DICOM App Services
+
+For both your FHIR and DICOM servers, you will set the Audience and Authority for Authentication. Follow the instructions below for each server, once for your Medical Imaging Server for DICOM and once for your FHIR Server.
+
+1. Navigate to the App Service that you deployed to Azure.
+1. Select **Configuration** to update the **Audience** and **Authority**:
+    1. Set the **Application ID URI** from the App Service as the **Audience**.
+    1. **Authority** is whichever tenant your application exists in, for example: ```https://login.microsoftonline.com/<tenant-name>.onmicrosoft.com```.
+
+### Update Key Vault for DICOM Cast
+
+1. Navigate to the DICOM Cast Key Vault that was created when you deployed DICOM Cast.
+1. Select **Access Policies** in the menu bar and click **Add Access Policy**.
+    1. Under **Configure from template**, select **Secret Management**.
+    1. Under **Select principal**, click **None selected**. Search for your Service Principle, click **Select** and then **Add**. 
+    1. Select **Save**.
+1. Select **Secrets** in the menu bar and click **Generate/Import**. Use the tables below to add secrets for your DICOM and FHIR servers. For each secret, use the **Manual Upload option** and click **Create**:
+
+#### Medical Imaging Server for DICOM Secrets
+
+| Name | Value |
+| :------- | :----- |
+| DicomWeb--Endpoint | ```<dicom-server-url>``` |
+| DicomWeb--Authentication--Enabled | "true" |
+| DicomWeb--Authentication--AuthenticationType | "ManagedIdentity" |
+| DicomWeb--Authentication--ManagedIdentityCredential--Resource | ```<dicom-server-url>``` |
+
+#### FHIR Server Secrets
+
+| Name | Value |
+| :------- | :----- |
+| fhir--Endpoint | ```<fhir-server-url>``` |
+| fhir--Authentication--Enabled | "true" |

```

Comment:
I don't know if these are case sensitive, but we should go ahead and update the `fhir` -> `Fhir`